deploy/vmwareuag.sh

```diff
@@ -0,0 +1,105 @@
+#!/usr/bin/env sh
+
+# Script for acme.sh to deploy certificates to a VMware UAG appliance
+#
+# The following variables can be exported:
+#
+# export DEPLOY_VMWAREUAG_USERNAME="admin"
+# export DEPLOY_VMWAREUAG_PASSWORD=""        - required
+# export DEPLOY_VMWAREUAG_HOST=""            - required (space seperated list) host:port
+# export DEPLOY_VMWAREUAG_HTTPS_INSECURE="1" - defaults to insecure
+#
+#
+
+########  Public functions #####################
+
+#domain keyfile certfile cafile fullchain
+vmwareuag_deploy() {
+  _cdomain="$1"
+  _ckey="$2"
+  _ccert="$3"
+  _cca="$4"
+  _cfullchain="$5"
+
+  # Some defaults
+  DEPLOY_VMWAREUAG_USERNAME_DEFAULT="admin"
+  DEPLOY_VMWAREUAG_HTTPS_INSECURE="1"
+
+  if [ -f "${DOMAIN_CONF}" ]; then
```

Comment:
no,  you don't need to import DOMAIN_CONF
just do your job.

Reply:
ack